ou_dedetai/wine.py

```diff
@@ -434,11 +434,21 @@ def set_win_version(exe, windows_version):
         wait_pid(process)
 
 
-def install_icu_data_files(app=None):
+def enforce_icu_data_files(app=None):
```

Comment:
Any reason for the name change?

Reply:
Yup, now it's called all the time rather than conditionally.

It'll break early if it's already installed, hence the install name is no longer strictly accurate